app/sample_manifest_excel/sample_manifest_excel/upload/processor/tube_rack.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'net/http'
 require 'json'
+require 'csv_parser_client'
```

Comment:
I guess you shouldn't add this line unless you're removing the 'include' one further down...

Reply:
Yes, changing it